.releaserc.json

```diff
@@ -0,0 +1,41 @@
+{
+    "branches": [
+      "master"
+    ],
+    "ci": false,
+    "plugins": [
+      [
+        "@semantic-release/commit-analyzer",
+        {
+          "preset": "conventionalcommits"
+        }
+      ],
+      [
+        "@semantic-release/release-notes-generator",
+        {
+          "preset": "conventionalcommits"
+        }
+      ],
+      [
+        "@semantic-release/github",
+        {
+          "successComment": "This ${issue.pull_request ? 'PR is included' : 'issue has been resolved'} in version ${nextRelease.version} :tada:",
+          "labels": false,
+          "releasedLabels": false
+        }
+      ],
+      [
+        "@semantic-release/exec",
+        {
+          "prepareCmd": "echo -n '${nextRelease.version}' > version && make generate-all"
+        }
+      ],
```

Comment:
The exec step will definitely need to be different for this repo--we might be able to run the existing goreleaser process?--but I think it'll be easier to get that wired up if we get the initial workflow merged in first.

Reply:
Cool! I need someone to merge this though, I don't have the write access here. @displague